jdaviz/configs/cubeviz/plugins/viewers.py

```diff
@@ -249,7 +249,7 @@ def get_sonified_cube(self, sample_rate, buffer_size, device, assidx, ssvidx,
 
         # Create a new entry for the sonified layer in uuid_lookup. The value is a dictionary
         # containing (x_size * y_size) keys with values being arrays that represent sounds
-        data_name = str(uuid.uuid4())
+        data_name = str(uuid.uuid4()) if results_label is None else results_label
```

Comment:
any reason to not just require passing `results_label`?

Reply:
Done